hack/make.ps1

```diff
@@ -0,0 +1,61 @@
+$DOCKER_PKG='github.com/docker/docker'
+$ErrorActionPreference = "Stop"
+
+# convert the .go-autogen bash script to ps1
+function ConvertGoAutogen {
+Write-Host "Converting hack\make\.go-autogen to hack\make\.go-autogen.ps1"
+(Get-Content hack\make\.go-autogen) `
```

Comment:
After looking at it, I think so. I started with a few patches, but ended up with more and more regexes to make it work.